docs/sphinx_docs/Input_file_formats/SMT-LIB2/index.md

```diff
@@ -0,0 +1,22 @@
+
+# SMT-LIB Version 2
+
+Alt-Ergo has partial support for the [SMT-LIB
+standard](http://smtlib.cs.uiowa.edu/papers/smt-lib-reference-v2.6-r2017-07-18.pdf)
+language from the SMT community.
+
+*Note*: As of version 2.5.0, enhanced support for the SMT-LIB language is
+provided by the new [Dolmen](http://gbury.github.io/dolmen/) frontend. To use
+it, pass the option `--frontend dolmen` to Alt-Ergo. This will become the
+default in a future release.
+
```

Comment:
I wad doing the same doc...

Reply:
The BV primitives are only available using the Dolmen frontend, I had to mention it. I can remove that paragraph.